docs/modules/ROOT/pages/reference/index.adoc

```diff
@@ -56,9 +56,85 @@
 |Stable
 |1.5.11
 |
+
 // standards: END
 |===
 
+[[unsupported]]
+== Unsupported Modules
+
+|===
+| Quarkus CXF extension | Alternative
+
+| JAX-RS +
+`cxf-rt-frontend-jaxrs` `cxf-rt-rs-client`
+| https://quarkus.io/guides/rest-json[Use quarkus core rest-json support] 
+
+| https://cxf.apache.org/fediz.html[Fediz]
+| legacy SAML protocol is not supported but openId connect is supported on Quarkus Core
```

Comment:
```suggestion
| Use Quarkus OpenID Connect
```

Which link could we put here? Is https://quarkus.io/guides/security-oidc-bearer-token-authentication-tutorial sufficient? Or is this also meant for client?

Reply:
best is https://quarkus.io/guides/security-oidc-code-flow-authentication-tutorial I think